Bukkit/src/main/java/com/github/intellectualsites/plotsquared/bukkit/listeners/PlayerEvents.java

```diff
@@ -268,53 +278,6 @@ public static boolean checkEntity(Entity entity, Plot plot) {
 
     @EventHandler public void onRedstoneEvent(BlockRedstoneEvent event) {
         Block block = event.getBlock();
-/*        switch (block.getType()) {
```

Comment:
Unrelated to this PR, but why was this commented out?

Reply:
I'm not sure why. I didn't mean to commit this code in this PR.